backend/__tests__/posts.js

```diff
@@ -397,6 +452,64 @@ describe('Posts', () => {
             })
         })
 
+        describe('when user unlike post in a valid way', () => {
```

Comment:
Move all the tests that you added into a separate describe() where you indicate the path of the request.

Reply:
Thanks for the review and the comment. 
I have just made the change !